public/locales/en/videocall.json

```diff
@@ -19,5 +19,18 @@
             "denied": "The camera and microphone permission has been denied.",
             "waiting": "The camera and microphone permission is needed so you contact can see and hear you."
         }
+    },
+    "errors": {
+        "title": "An error has appeared",
+        "desc": "An error has appeared that will probably prevent you to see your loved one, here is the detail of the error:",
```

Comment:
"An error occured and will probably prevent you from seeing your loved one. Detail:"